src/openrct2/ride/TrackPaint.h

```diff
@@ -28,6 +30,21 @@ extern const uint8_t edges_4x4[];
 
 extern const uint8_t track_map_1x4[][4];
 
+namespace TrackPaint::Flat
+{
+    // Lift and No lift sprites
+    using BasicSprites = std::array<std::array<uint16_t, NumOrthogonalDirections>, 2>;
+
+    // No lift sprites
+    using NoLiftSprites = std::array<uint16_t, NumOrthogonalDirections>;
+
+    // Lift and No Lift (sprites * 2 {front, back})
+    using ComplexSprites = std::array<std::array<std::array<uint16_t, 2>, NumOrthogonalDirections>, 2>;
+
+    // Inverted and Not Inverted (Lift and No Lift) sprites
```

Comment:
Needs a bit of clarification as to what is the outer and what in the inner array.

Reply:
Yes thats a good idea I'll make it nested BasicSprites.